### PR TITLE
Force full omegac0 decay chain

### PR DESCRIPTION
**Pull request by ZFederica**

### CODE REVIEW

**Comment by lvermunt** [comment]
Hi @chiarazampolli, could you perhaps approve this PR? As Gustavo wrote, this is needed for ALIROOT-8813 (not 8826, that was probably a typo). Thanks!

**Review by chiarazampolli — APPROVED** [review]

File: PYTHIA8/AliPythia8/AliDecayerPythia8.cxx [hunk 1]
```diff
@@ -768,7 +768,13 @@ void AliDecayerPythia8::ForceHadronicD(Int_t optUse4Bodies, Int_t optUseDtoV0, I
         fPythia8->ReadString("4332:onIfMatch = 3334 -11 12");
     }
     else if (optForceOmegacChannel == 2){ //Omegac0 -> Xi Pi
-	fPythia8->ReadString("4332:onIfMatch = 3312 211");
+		fPythia8->ReadString("4332:onIfMatch = 3312 211");
```
[the remark this answers is not in the document]
Comment (ZFederica) [reply PYTHIA8/AliPythia8/AliDecayerPythia8.cxx]: Ok

**Comment by chiarazampolli** [comment]
Hello @TimoWilken , @ZFederica ,
I cannot find the error in this build, maybe you can also check? 
Chiara

**Comment by TimoWilken** [comment]
Hi @chiarazampolli, as far as I know this is a long-standing problem with floating-point exception handling in AliRoot that surfaces in the "AliRoot-guntest" checks. If you want me to force-merge one of the PRs, let me know.

**Comment by gconesab** [comment]
> Hi @chiarazampolli, as far as I know this is a long-standing problem with floating-point exception handling in AliRoot that surfaces in the "AliRoot-guntest" checks. If you want me to force-merge one of the PRs, let me know.

Hi @TimoWilken, I would force all the commits that fail at this floating point step, if others do not oppose.
Cheers ... Gustavo

**Pull request merged**